_couches/kevin&sarah.md

```diff
@@ -0,0 +1,16 @@
+---
+github: kevinmcalear
+twitter: kevinmcalear
+email: kevin.mcalear@gmail.com
+website: http://kevinmcalear.com/
+facebook: KPMcAlear
+city: Seattle
+country: USA
```

Comment:
Just realized another issue might be with the filename. /couches/kevin&sarah.html might be troublesome?

Let me check it out locally.